.circleci/config.yml

```diff
@@ -443,6 +443,18 @@ jobs:
                 paths:
                     - ~/.cache/yarn
                     - node_modules
+            - run:
+                command: tools/parse-release-notes parse --android --destination android/fastlane/metadata/android/de-DE/changelogs/default.txt --language de --production
+                name: Parse german android release notes
+            - run:
+                command: tools/parse-release-notes parse --android --destination android/fastlane/metadata/android/en-GB/changelogs/default.txt --language en --production && tools/parse-release-notes parse --android --destination android/fastlane/metadata/android/en-US/changelogs/default.txt --language en --production
+                name: Parse english android release notes
+            - run:
+                command: tools/parse-release-notes parse --ios --destination ios/fastlane/metadata/de-DE/release_notes.txt --language de --production
+                name: Parse german ios release notes
+            - run:
+                command: tools/parse-release-notes parse --ios --destination ios/fastlane/metadata/en-US/release_notes.txt --language en --production
+                name: Parse english ios release notes
```

Comment:
didn't you have a fallback to use the default language in the release notes if there are no release notes in the specified language? So what I want to now is whether it can happen that there are e.g two english release nodes or sth.

Reply:
Not exactly sure if I understand your question correctly. If release notes for e.g. `ar` are created but there are none in that language, it will be completely in english. That should not be a problem though as the fallback/default in the stores is english anyway.

Reply:
We are not sure how release notes are handled by the stores. We will see :shrug:

Reply:
Yes :D